charts/generic-govuk-app/templates/cron-task.yaml

```diff
@@ -12,6 +12,7 @@ metadata:
     app.kubernetes.io/component: "{{ .name }}"
 spec:
   schedule: "{{ .schedule }}"
+  {{ if .timeZone }}timeZone: "{{ .timeZone }}"{{- end }}
```

Comment:
i'm not particularly familiar with the syntax here but do we need the string wrapping the .timeZone variable. i.e. could 

```
{{ if .timeZone }}timeZone: "{{ .timeZone }}"{{- end }}
```

be

```
{{ if .timeZone }}timeZone: {{ .timeZone }}{{- end }}
```

Reply:
So if you do that you lose the previous wrapping.

E.g. with the former:

```
➜  app-config git:(cron-timezone) ✗ cat /tmp/test-2.helm | grep timeZone
  timeZone: "Europe/London"
  timeZone: "Europe/London"
  timeZone: "Europe/London"
```

with the latter:

```
➜  app-config git:(cron-timezone) ✗ cat /tmp/test-2.helm | grep timeZone
  timeZone: Europe/London
  timeZone: Europe/London
  timeZone: Europe/London
```

so it feels safer to do so.

Reply:
oh right thanks. I hadn't clocked it stripped the wrapping `"`s. Thanks for the explanation